ui/apps/platform/src/types/vulnerability.proto.ts

```diff
@@ -1,39 +1,85 @@
-import type { CVSSV2, CVSSV3, VulnerabilitySeverity, VulnerabilityState } from './cve.proto';
```

Comment:
Could you explain the motive behind the type changes?

Reply:
Yeah the previous code in this file was actually dead code not being used. All the type changes added in this PR should match the proto definitions 1:1. It's another topic I added to our office hours for next week.